Upload A File/Main.gcomp/File Download or Upload.gviweb

```diff
@@ -1,13 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<SourceFile Checksum="A3B6DB1C934DEA33B7552B44D9554FF5AC36AF6057D36E90B7073B31F4A3E51D55D0998DEAF62FCEE690C891EBDAA85674DC58B4036163F2CB4AF393A592F855" Timestamp="1D530FD0DCB8715" xmlns="http://www.ni.com/PlatformFramework">
+<SourceFile Checksum="A2C7F74274CCB1694BB216F5F075F46D009C1BF79A2DBCDF2B76C6784E8A99B426D85B76799B16E43E686F55C58B1960AC4AE2DA1C56D419382B94B3D073D31D" Timestamp="1D53EBD52AB304F" xmlns="http://www.ni.com/PlatformFramework">
```

Comment:
Updated the example to be event based

![image](https://user-images.githubusercontent.com/1588923/61586273-3b6b1280-ab35-11e9-8184-9c00002f16bc.png)

Avoid polling loops. Note the upload loop is not polling, the node blocks until a file is selected. Think how DAQmx read blocks and waits for data. It actually might be a good idea to add timeout support. Could be a good exercise ;)

This screenshot shows how the class is added:

![image](https://user-images.githubusercontent.com/1588923/61586288-a1579a00-ab35-11e9-93d4-7771c5985fbd.png)